spec/yelp/burst_struct_spec.rb

```diff
@@ -1,155 +0,0 @@
-require 'yelp/burst_struct'
-
-describe BurstStruct::Burst do
-  describe '#keys' do
-    subject(:struct) { BurstStruct::Burst.new(foo: 'bar', baz: 'qux') }
-
-    it 'should return' do
-      expect(struct.keys).to eql [:foo, :baz]
-    end
-  end
-
-  describe '#foo' do
-    subject(:struct) { BurstStruct::Burst.new(foo: 'bar') }
-
-    context 'when a key exists' do
-      it 'should return' do
-        expect(struct.foo).to eql 'bar'
-      end
-
-      it { should have_key(:foo) }
-      it { should have_key('foo') }
-
-      it { should respond_to(:foo) }
-    end
-
-    context 'when a key does not exist' do
-      it 'should not respond to it' do
-        expect(struct.respond_to? :super_foo).to eql false
-      end
-
-      it { should_not have_key(:super_foo) }
-      it { should_not have_key('super_foo') }
-    end
-  end
-
-  context 'n deep nested hash' do
-    let(:hash) { Hash[ futurama: { characters: { robots: { best: 'bender' } } } ] }
-
-    subject(:struct) { BurstStruct::Burst.new(hash) }
-
-    describe '#foo#bar#baz#biz' do
-      it 'should return' do
-        expect(struct.futurama.characters.robots.best).to eql 'bender'
-      end
-    end
-  end
-
-  context 'arrays' do
-    let(:hash) { Hash[ businesses: [ { name: 'Yelp', location: 'San Francisco' },
-                                     { name: 'Ice Cream', flavor: 'Chocolate' },
-                                     [ { name: 'Moe', occupation: 'Bartender' } ] ] ] }
-
-    subject(:struct) { BurstStruct::Burst.new(hash) }
-
-    describe '#businesses[0].name' do
-      it 'should return' do
-        expect(struct.businesses[0].name).to eql 'Yelp'
-        expect(struct.businesses[0].location).to eql 'San Francisco'
-      end
-    end
-
-    describe '#business[1].name' do
-      it 'should return' do
-        expect(struct.businesses[1].name).to eql 'Ice Cream'
-        expect(struct.businesses[1].flavor).to eql 'Chocolate'
-      end
-    end
-
-    describe 'nested arrays' do
-      it 'should parse arrays all the way down' do
```

Comment:
@mittonk also the endpoint specs do take the json and parse them into the response objects and models to verify that they work correctly